testutil/test_helper.go

```diff
@@ -366,6 +369,33 @@ func (h *TestHelper) HTTPGetURL(url string) (string, error) {
 	return body, err
 }
 
+// RunLogsAndEventTests runs the TestLogs and TestEvents tests in a separate
+// process
+func (h *TestHelper) RunLogsAndEventTests() {
+	// don't run for tests that were singled out
+	if h.run != "" {
+		return
+	}
+	args := []string{"test", "-v", ".",
+		"-integration-tests",
+		"-linkerd", h.linkerd,
+		"-run",
+	}
+	out, err := exec.Command("go", append(args, "TestLogs")...).Output()
```

Comment:
Why do we invoke these tests by shelling out instead of calling the test functions directly?

Reply:
From within `TestMain` there are no references to `t`, so I couldn't find a way to call a test function directly. I would be glad to find there's a better way than shelling out.